Formula/o/overarch.rb

```diff
@@ -0,0 +1,58 @@
+class Overarch < Formula
+  desc "Data driven description of software architecture"
+  homepage "https://github.com/soulspace-org/overarch"
+  url "https://github.com/soulspace-org/overarch/archive/refs/tags/v0.14.0.tar.gz"
+  sha256 "7ba86645fa2cf9893a9c9c2eda416d4f4ca0b2bcbde683b4b7b0fc72b9bca909"
+  license "EPL-1.0"
+  head "https://github.com/soulspace-org/overarch.git", branch: "main"
+
+  depends_on "leiningen" => :build
+  depends_on "openjdk"
+
+  resource "jar" do
+    url "https://github.com/soulspace-org/overarch/releases/download/v0.14.0/overarch.jar"
+    sha256 "7d573e0c044e63487cf9104142651e79beb849282916cf63dcb3a3256211bff0"
```

Comment:
Why not use this as the URL?

Reply:
I assume you mean why not lift that `url` out of the `jar` block into the main definition. That would preclude being able to build this formula from source via `lein uberjar`, which is what I was thinking of adding next.

Reply:
Do you have any thoughts on that?

Reply:
Then what is the jar for? I'm confused, that seems like a duplicate install

Reply:
See 138938139e6 for changes to support building this formula from upstream's HEAD commit.

Reply:
I hope that makes more sense now. Let me know if you have any further feedback or need clarification.

Reply:
🤦 Ah, I see what you mean now. Sorry for missing that earlier. I'll resolve it now.

Reply:
Should be okay now.